### ISSUE TITLE
adding dependency to docconv on go 1.11 fails

### ISSUE THREAD

**Issue by rafalkrupinski** [sajari/docconv#55]
```shell
$ go get code.sajari.com/docconv/
go: finding code.sajari.com/docconv v1.0.0
go: downloading code.sajari.com/docconv v1.0.0
go: finding github.com/JalfResi/justext latest
go: downloading github.com/JalfResi/justext v0.0.0-20170829062021-c0282dea7198
go: finding github.com/levigross/exp-html latest
go: downloading github.com/levigross/exp-html v0.0.0-20120902181939-8df60c69a8f5
build code.sajari.com/docconv: cannot find module for path code.google.com/p/go.net/html
$
```

**Comment by rafalkrupinski** [comment]
I see this is more or less fixed in master. Would you be so kind and make a release tag, either from master, or fixed v1.0.1 or both?

**Comment by mish15** [comment]
Good idea. Done!
